docs/root/version_history/current.rst

```diff
@@ -52,6 +52,7 @@ Removed Config or Runtime
 New Features
 ------------
 
+* crash support: restore crash context when processing requests or responses due to a filter's callback if the callback doesn't already restore the context.
```

Comment:
Code looks good, but I would really like both here and in interface and other comments below to be more crisp on what a "filter's callback" is. I think this is one of these things that if you've spent a lot of time looking at the problem it's obvious, but to me, there are many callbacks that exist for filters; filters implement callback functions, they invoke callbacks, etc. I think giving an "e.g. foo" would help.

FWIW I think I have some intuition here based on our conversation the other day, but a random reader would not.

Reply:
Done.